tests/property/ic_api/canister_version/test/tests.ts

```diff
@@ -65,12 +67,15 @@ export function getTests(): Test {
 
 async function checkInitVersion(
     actor: Actor,
-    expectedVer: bigint
+    expectedVer: [bigint] | []
```

Comment:
Let's not use Candid JS options unless we have to

Reply:
Done